README.md

```diff
@@ -230,6 +230,25 @@ returned, only the diff.
 
 Currently accepts the same options as the read stream except for reverse.
 
+#### `watcher = db.watch([range], [onchange])`
+
+Listens to changes that are on the optional `range`.
+
+`range` options are the same as `createReadStream` except for reverse.
+
+`watcher.destroy()`\
+Closes the watcher.
+
+`watcher.on('change', (newVersion, oldVersion) => {})`\
+Emitted after a feed change.
+
+`watcher.on('error', onerror)`\
+Critical and unexpected errors will be thrown, but watcher is normally graceful.\
+Also, watcher auto closes on errors.
+
+`watcher.on('close', onclose)`\
+Emitted after the watcher is closed.
```

Comment:
why the escaped new lines?

Reply:
On this case, with escape vs without vs with break line:

![image](https://user-images.githubusercontent.com/12686176/222517916-5f28e08f-dfad-4976-8933-8df0995effdf.png)

=>

![image](https://user-images.githubusercontent.com/12686176/222517734-28e274fe-1180-4832-9a59-aac99a5a5b0e.png)

Reply:
I don't like the extra space in between that a normal break line does, so that's why I escape them, but I can remove the escapes if you like